### PR TITLE
[DYN-7334] Revised TuneUp controls layout

### PR DESCRIPTION
**Pull request by ivaylo-matov**
### Purpose

PR to address [DYN-7334](https://jira.autodesk.com/browse/DYN-7334)

Revised layout as in [Figma ](https://www.figma.com/design/tEYV4LQG8lKUqNf5jpgrQO/TuneUp?node-id=196-3000&node-type=canvas&t=z9C0eqb1JUyYIXoP-0)

- Profiled nodes are split into separate DataGrids (LatestRun, PreviousRun, NotExecuted).
- Renamed node indicator with an added tooltip.
- Added export dropdown button for CSV and JSON formats.
- Group colors in the UI now reflect the group background colors in the workspace.
- Node name, group color, and group name dynamically update in the UI.
- Group total times are shown as a separate row in the UI.
- Columns can be sorted by clicking on headers; "Not Executed" is sorted only by name.
- Total times are based on rounded node execution times.

DYN-7334-ProgressRecord

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB

### Reviewers
@QilongTang
@reddyashish

### FYIs
@Amoursol
@dnenov

### CODE REVIEW

File: TuneUp/ProfiledNodeViewModel.cs [hunk 1]
```diff
@@ -7,18 +7,116 @@
...
+                return originalName;
+            }
+            internal set
+            {
+                if (originalName == value) return;
+                originalName = value;
+                RaisePropertyChanged(nameof(OriginalName));
+            }
+        }
+        public bool IsGroupExecutionTime
```
Comment (reddyashish) [inline TuneUp/ProfiledNodeViewModel.cs:61]: Add summary to this public property

File: TuneUp/TuneUpViewExtension.cs [hunk 2]
```diff
@@ -18,7 +18,6 @@ public class TuneUpViewExtension : ViewExtensionBase, IViewExtension
 
         public override void Dispose()
         {
-            TuneUpView.Dispose();
```
Comment (reddyashish) [inline TuneUp/TuneUpViewExtension.cs:21]: Any reason to removing the dispose on the view when the extension is closed?

Comment (ivaylo-matov) [reply TuneUp/TuneUpViewExtension.cs:21]: AFAICT it was only unsubscribing from DynamoWindow_SizeChanged, which is not needed anymore.

File: TuneUp/ProfiledNodeViewModel.cs (hunk 1 above)
Comment (ivaylo-matov) [reply TuneUp/ProfiledNodeViewModel.cs:65]: done

**Review by reddyashish — APPROVED** [review]

**Pull request merged**